Bindings/Java/tests/TestEditProperties.java

```diff
@@ -38,6 +38,23 @@ public static void main(String[] args) {
                     System.exit(-1);
                 } 
         }
+		// Now edit a Vec6 Property (inertia)
+		AbstractProperty inertiaProp = bdy.getPropertyByName("inertia");
+		Vec6 oldInertia = new Vec6();
+		for(int i=0; i<6; i++){
```

Comment:
`PropertyHelper.getValueVec6(inertiaProp, i)` also appears on L53; what is L45 testing that L53 isn't?